decidim-core/app/forms/decidim/amendable/form.rb

```diff
@@ -46,7 +46,7 @@ def normalized_body(resource)
 
       # Validates the emendation using the amendable form.
       def amendable_form_must_be_valid
-        parse_hashtaggable_params
+        parse_hash_params
```

Comment:
I don't like the `parse_hash_params` name, as it doesn't correspond with the new usage (where's the hash??). Maybe something more appropriate like `parse_conent_params`? 
```suggestion
        parse_content_params
```

Reply:
Fine by me.